src/layouts/Header.tsx

```diff
@@ -268,7 +271,7 @@ class Header extends React.Component<IProps, null> {
                 </div>
               </span> : <span></span>
             }
-            {!currentAccountAddress ?
+            { !currentAccountAddress || currentAccountAddress === "" ?
```

Comment:
same here, "" is always falsey

Reply:
This is checking for empty strings, and is not always falsey. Please see this demonstration proving such: 
![image](https://user-images.githubusercontent.com/5522128/82394979-75889780-9a18-11ea-8461-d80f225263e8.png)

Reply:
Your example is testing the wrong logic.  We're talking here about how `!currentAccountAddress` evaluates.

![image](https://user-images.githubusercontent.com/1821666/82565547-9f40dc00-9b48-11ea-8c6d-892e504a3ba6.png)